routers/api/v1/swagger/repo.go

```diff
@@ -351,3 +351,27 @@ type swaggerRepoCollaboratorPermission struct {
 	// in:body
 	Body api.RepoCollaboratorPermission `json:"body"`
 }
+
+// swagger:response Project
+type swaggerProject struct {
+	// in:body
+	Body api.Project `json:"body"`
+}
+
+// swagger:response ProjectList
+type swaggerProjectList struct {
+	// in:body
+	Body []api.Project `json:"body"`
+}
+
+// swagger:response ProjectBoard
+type swaggerProjectBoard struct {
+	// in:body
+	Body api.ProjectBoard `json:"body"`
+}
+
+// swagger:response ProjectBoardList
```

Comment:
```suggestion
// Project
// swagger:response Project
type swaggerProject struct {
	// in:body
	Body api.Project `json:"body"`
}

// ProjectList
// swagger:response ProjectList
type swaggerProjectList struct {
	// in:body
	Body []api.Project `json:"body"`
}

// ProjectBoard
// swagger:response ProjectBoard
type swaggerProjectBoard struct {
	// in:body
	Body api.ProjectBoard `json:"body"`
}

// ProjectBoardList
// swagger:response ProjectBoardList
```
make it similar to others ...